services/scabbard/cli/man/scabbard-contract.1.md

```diff
@@ -0,0 +1,44 @@
+% SCABBARD-CONTRACT(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**scabbard-contract** — Provides contract management functionality.
```

Comment:
Per man page convention, the description under NAME should not end with a period.

Reply:
Fixed